Core/Helper.php

```diff
@@ -9,18 +9,15 @@
 
 namespace Wirecard\Oxid\Core;
 
+use DateTime;
+use Exception;
```

Comment:
[minor] Maybe add a newline to separate from the following group.

Reply:
Currently we only enforce an empty line if there is a base path and it changes from one line to another, for example:

```php
use Wirecard\Oxid\Model\PaymentMethod;
use Wirecard\Oxid\Model\Transaction;

use OxidEsales\Eshop\Core\Registry;
use OxidEsales\Eshop\Core\Exception\StandardException;

use DateTime;
use Exception;
```

This is also implemented in the PHPCS ruleset.

Reply:
👍